### PR TITLE
remove related-tools page and links to it, point to Awesome Coq instead

### PR DESCRIPTION
**Pull request by palmskog**
The "Related Tools" page is completely out of date with respect to modern tools for Coq and built on Coq. I think the [Awesome Coq](https://github.com/coq-community/awesome-coq) project is now in such a good shape (100+ entries, 100+ GitHub stars) that it can take over the role of Related Tools completely. Hence, this PR removes Related Tools and all links to it, and points to Awesome Coq where appropriate.

cc: @Zimmi48 @anton-trunov

### CODE REVIEW

**Comment by Zimmi48** [comment]
I'm supporting of the idea. The Related Tools page has always been a hell to maintain and recently we have been talking of removing it and linking to the wiki. I think that linking to awesome-coq is a better choice.

File: pages/index.html [hunk 1]
```diff
@@ -61,10 +61,9 @@
 <div class="frameworklabel"><a style="color:black;font-weight:bold" href="/community.html">User Community</a></div>
 <div class="frameworkcontent">
 
-<p>Coq has a large and active user community. They contribute
-formal developments, Coq extensions
-(see <a href="/packages.html">Coq Package Index</a>) and tools based
-on Coq (see <a href="/related-tools.html">Related Tools</a>).  We have
+<p>Coq has a large and active user community. They contribute formal
+developments, Coq extensions, and tools based on Coq (see the
+<a href="/packages.html">Coq Package Index</a>). We have
```
Comment (Zimmi48) [inline pages/index.html:66]: Sounds like it would be worth mentioning Awesome Coq on the index page as well.

**Review by Zimmi48 — COMMENTED** [review]
Please add a redirection from Related Tools to Awesome Coq. See https://github.com/coq/www/blob/master/aliases.footer.conf.

**Comment by palmskog** [comment]
> Please add a redirection from Related Tools to Awesome Coq. See https://github.com/coq/www/blob/master/aliases.footer.conf.

I added a redirect to the tools section of Awesome Coq in ab67226. I can't test locally if it works, but I tried to follow the pattern from Nahas' tutorial.

**Review by Zimmi48 — APPROVED** [review]

**Pull request merged**